init-pab.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
You probably didn't commit this file with all the absolute paths on purpose did you? Same for the script below

Reply:
no, those will be deleted